### PR TITLE
No tests with admin user

### PR DESCRIPTION
**Pull request by hmunfru**
### REVIEWER

@jesuspg 
### DESCRIPTION
- Check if the user has admin role
- Delete only ports for tenant-id
- Test only RegionONe region
  ### TEST
  Done

### CODE REVIEW

File: fiware-region-sanity-tests/commons/fiware_cloud_test_case.py [hunk 1]
```diff
@@ -182,6 +184,9 @@ def init_clients(cls, tenant_id, test_flavor, test_image):
                                                    auth_session=cls.auth_sess)
         cls.neutron_operations = FiwareNeutronOperations(cls.logger, cls.region_name, tenant_id,
                                                          auth_session=cls.auth_sess)
+        cls.keystone_operations = FiwareKeystoneOperations(cls.logger, cls.region_name, tenant_id, user_id=cls.conf[PROPERTIES_CONFIG_CRED][PROPERTIES_CONFIG_CRED_USER],
```
Comment (jesuspg) [inline fiware-region-sanity-tests/commons/fiware_cloud_test_case.py]: line too long

File: fiware-region-sanity-tests/commons/keystone_operations.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        """
+        It checks the roles associated to the user in the project. In case it is admin, it
+        exits the execution.
+        :return: nothing
+        """
+        roles= self.client.roles.list(user=self.user_id, project=self.tenant_id)
+        for role in roles:
+            if "admin" in role.name:
+                self.logger.debug( "User with role admin cannot be used")
+                exit()
```
Comment (jesuspg) [inline fiware-region-sanity-tests/commons/keystone_operations.py]: change exit by assert fail

File: fiware-region-sanity-tests/commons/fiware_cloud_test_case.py [hunk 3]
```diff
@@ -389,30 +394,34 @@ def reset_world_ports(cls, world, suite=False):
...
-
+        ports = []
         if suite and cls.with_networks:
             # get pre-existing port list (ideally, empty when starting the tests)
             try:
                 port_list = cls.neutron_operations.list_ports()
                 for port in port_list:
                     cls.logger.debug("init_world() found port '%s' not deleted", port['id'])
                     world['ports'].append(port['id'])
+                    ports.append(port)
```
Comment (flopezag) [inline fiware-region-sanity-tests/commons/fiware_cloud_test_case.py]: Why we need to keep world['ports'] and ports?

File: fiware-region-sanity-tests/commons/keystone_operations.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+
+from keystoneclient import client
+from commons.constants import DEFAULT_REQUEST_TIMEOUT
+
+class FiwareKeystoneOperations:
+
+    def __init__(self, logger,  region_name,tenant_id, user_id, **kwargs):
```
Comment (flopezag) [inline fiware-region-sanity-tests/commons/keystone_operations.py]: coding style...

File: fiware-region-sanity-tests/commons/neutron_operations.py [hunk 5]
```diff
@@ -264,12 +264,12 @@ def find_networks(self, **kwargs):
         """
         found = []
         search = [(key.replace('router_', 'router:'), value) for (key, value) in kwargs.items()]
-        network_list = self.client.list_networks(retrieve_all=True).get('networks')
-        for network in network_list:
+        list = self.client.list_networks(retrieve_all=True).get('networks')
```
Comment (flopezag) [inline fiware-region-sanity-tests/commons/neutron_operations.py]: I should not change the name of the variables in this case. I think that the semantic of the previos variables is better that the new ones.

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**